extensions/eclipse-che-theia-plugin-remote/src/node/plugin-remote-init.ts

```diff
@@ -122,15 +149,16 @@ export class PluginRemoteInit {
             return originalStart.call(this, params);
         };
 
+        this.initWebsocketServer(inversifyContainer.get(ILoggerServer));
```

Comment:
if you need stuff from inversifyContainer let's make it private variable of the class instead of giving to arguments to local methods

Reply:
What is the advantage?

Reply:
you're passing in tons of method a logger while you're always in the same class and the attribute is local, so no need to add extra parameters.

Reply:
Passing it as an argument constrains the visiblity of the logger server. I don't think passing parameters is a bad thing in itself, as long as we don't have too many.

Reply:
BTW the whole stuff should be reworked while here it was just a tiny remark.